anta/result_manager/__init__.py

```diff
@@ -137,6 +137,11 @@ def results_by_status(self) -> dict[AntaTestStatus, list[TestResult]]:
         """A cached property that returns the results grouped by status."""
         return {status: [result for result in self._result_entries if result.result == status] for status in AntaTestStatus}
 
+    @cached_property
+    def results_by_category(self) -> list[TestResult]:
```

Comment:
Will need to clear it in the `add()` method.

Reply:
Thanks!